platform/src/main/java/org/conscrypt/Platform.java

```diff
@@ -20,10 +20,10 @@
 import static android.system.OsConstants.SO_SNDTIMEO;
 
 import android.system.ErrnoException;
-import android.system.Os;
 import android.system.StructTimeval;
 import dalvik.system.BlockGuard;
 import dalvik.system.CloseGuard;
+import libcore.io.Libcore;
```

Comment:
Travis is complaining that this import is in the wrong place (should be next to the libcore.net import).

Reply:
Done, fixed the import order.